src/types/care_provider.ts

```diff
@@ -123,7 +123,7 @@ export type CareProvider = {
   substanceUse: {
     supported: boolean;
     duiSupported: boolean;
-    services: { [key in SubstanceUseServices]: boolean };
+    services: { [key in SubstanceUseServices]: boolean } & { PeerSupport: boolean };
```

Comment:
can we add a comment somewhere (here, or in the data parsing script, or both) to just explain in plain language why we're doing this (this = augmenting the substance use services with this separate field) so future developers will understand that this is intentional and a result to transform the data from it's original structure to better contextualize it for the end user

Reply:
Will do! I'll write the comment here.

Reply:
Added comment.